contractcourt/channel_arbitrator.go

```diff
@@ -2448,13 +2479,12 @@ func (c *ChannelArbitrator) channelAttendant(bestHeight int32) {
 			log.Tracef("ChannelArbitrator(%v) got new signal "+
 				"update!", c.cfg.ChanPoint)
 
-			// First, we'll update our set of signals.
-			c.htlcUpdates = signalUpdate.newSignals.HtlcUpdates
+			// We'll update the ShortChannelID.
```

Comment:
So we don't have any `HtlcUpdates` here?

Reply:
No this is just for updating the ShortChannelID and maybe some other non-htlc info in the future